lib/resources/dstu2/procedure.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Procedure
+field_name_base_url: http://hl7.org/fhir/DSTU2/procedure-definitions.html#Procedure
+fields:
+- name: code
+  type: CodeableConcept
+  binding:
+    description: A code to identify a specific procedure
+    terminology:
+    - display: Procedure Codes (SNOMED CT)
+      system: http://snomed.info/sct
+    - display: CPT
+      system: http://www.ama-assn.org/go/cpt
+    - display: ICD-10-PCS
+      system: http://www.icd10data.com/icd10pcs
+
+- name: performer
```

Comment:
This isn't getting used for the terminology binding table currently, do we need it?